tests/system/pages/modals/WalletModalWindow.ts

```diff
@@ -15,17 +17,19 @@ export class WalletModalWindow extends BaseModal<WalletModalElements> {
   }
 
   public async openWalletModalWindow(): Promise<WalletModalWindow> {
-    await this.previousPage.click(this.previousPage.pageElements.url);
+    await step('Open wallet modal window', async () => {
+      await this.previousPage.click(this.previousPage.pageElements.url);
+    });
 
     return this;
   }
 
-
   public async openMultisigModalWindow(): Promise<MultisigModalWindow> {
-    await this.page.getByRole('button', { name: 'Add' }).click();
-    await this.page.getByRole('menuitem', { name: 'Multisig' }).click();
+    await step('Open Multisig modal from Wallet modal', async () => {
```

Comment:
lets make it more generic
```suggestion
    await step('Open Multisig creation widget', async () => {
```